diff --git a/php_skywalking.h b/php_skywalking.h
index f3e966f..4b4849d 100644
--- a/php_skywalking.h
+++ b/php_skywalking.h
@@ -77,7 +77,8 @@ extern zend_module_entry skywalking_module_entry;
 #define SKY_IS_SWOFT(class_name, func_name) (SKY_STRCMP(class_name, "Swoft\\Http\\Server\\Swoole\\RequestListener") && SKY_STRCMP(func_name, "onRequest"))
 #define SKY_IS_TARS(class_name, func_name) (SKY_STRCMP(class_name, "Tars\\core\\Server") && SKY_STRCMP(func_name, "onRequest"))
 #define SKY_IS_LARAVELS(class_name, func_name) ((SKY_STRCMP(class_name, "Hhxsv\\LaravelS\\LaravelS") || SKY_STRCMP(class_name, "Hhxsv5\\LaravelS\\LaravelS")) && SKY_STRCMP(func_name, "onRequest"))
-#define SKY_IS_SWOOLE_FRAMEWORK(class_name, func_name) SKY_IS_HYPERF(class_name, func_name) || SKY_IS_SWOFT(class_name, func_name) || SKY_IS_TARS(class_name, func_name) || SKY_IS_LARAVELS(class_name, func_name)
+#define SKY_IS_SWOOLE_BUNDLE(class_name, func_name) (SKY_STRCMP(class_name, "K911\\Swoole\\Bridge\\Symfony\\HttpKernel\\HttpKernelRequestHandler") && SKY_STRCMP(func_name, "handle"))
+#define SKY_IS_SWOOLE_FRAMEWORK(class_name, func_name) SKY_IS_HYPERF(class_name, func_name) || SKY_IS_SWOFT(class_name, func_name) || SKY_IS_TARS(class_name, func_name) || SKY_IS_LARAVELS(class_name, func_name) || SKY_IS_SWOOLE_BUNDLE(class_name, func_name)
 
 #if PHP_VERSION_ID < 80000
 #define SKY_ZEND_CALL_METHOD(obj, fn, func, ret, param, arg1, arg2) zend_call_method(obj, Z_OBJCE_P(obj), fn, ZEND_STRL(func), ret, param, arg1, arg2);
